frontend/src/pages/landing/styles.module.scss

```diff
@@ -5,20 +5,20 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  width: 110%;
-  height: 110%;
-  padding: 0 60px;
-  overflow: hidden;
-  background-color: var(--dark-green-80);
-  background-image: url("@assets/img/background-leters.png"),
-    url("@assets/img/background-typewriter.jpg");
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 1440px, cover;
+  width: 100vw;
+  height: 100vh;
+  transform: translate(-50%, -50%);
+}
+
+.parallaxImg {
+  position: fixed;
+  top: 50%;
+  left: 50%;
   transform: translate(-50%, -50%);
 }
 
 .signLink {
+  z-index: 5;
```

Comment:
Let's use defined z-index from variables, or add this one to variables if necessary